configure.ac

```diff
@@ -0,0 +1,461 @@
+# Gotchas:
+#
+# 1. Quadrigraphs. M4 consumes a number of important special characters, so
+#    Autoconf uses 4-character sequences, e.g. "@%:@" is the octothorpe (#).
+#    See: https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Quadrigraphs.html
+#
+# 2. Booleans. The convention for Autoconf variables, which we follow, is
+#    "yes" for true and "no" for false. This differs from the Charliecloud
+#    convention of non-empty for true and empty for false.
+
+AC_INIT([Charliecloud], [m4_esyscmd_s([misc/version])],
+        [https://github.com/hpc/charliecloud])
+AC_PREREQ([2.69])
+AC_CONFIG_SRCDIR([bin/ch-run.c])
+AC_CONFIG_AUX_DIR([build-aux])
+
+AM_INIT_AUTOMAKE([1.15 -Wall -Werror foreign subdir-objects])
```

Comment:
`autoconf 1.13.4` is the newest version available on Centos7. I was able to successfully build by changing this to `1.13`.